### PR TITLE
* #1: Moved files

### LINKED ISSUE #1: Move Hive dialect to a new repo

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+addons:
+  sonarcloud:
+    organization: exasol
+
+install:
+    - travis_retry mvn dependency:resolve
+
+script:
+  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration verify sonar:sonar
```
Comment (chiaradiamarcelo) [inline .travis.yml:19]: the SONAR_TOKEN env variable is missing
```suggestion
    - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration verify sonar:sonar -Dsonar.login=${SONAR_TOKEN}
```

File: doc/user_guide/hive_user_guide.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+## Uploading the JDBC Driver to EXAOperation
+
+1. [Create a bucket in BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/create_new_bucket_in_bucketfs_service.htm)
+1. Upload the driver to BucketFS
+
+This step is necessary since the UDF container the adapter runs in has no access to the JDBC drivers installed via EXAOperation but it can access BucketFS.
+
+## Installing the Adapter Script
+
+Upload the latest available release of [Virtual Schema JDBC Adapter](https://github.com/exasol/virtual-schemas/releases) to Bucket FS.
```
Comment (chiaradiamarcelo) [inline doc/user_guide/hive_user_guide.md:37]:
```suggestion
Upload the latest available release of [Hive Virtual Schema](https://github.com/exasol/hive-virtual-schema/releases) to Bucket FS.
```

File: doc/user_guide/hive_user_guide.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+
+## Testing information
+
+In the following matrix you find combinations of JDBC driver and dialect version that we tested.
+The dialect was tested with the Cloudera Hive JDBC driver available on the [Cloudera downloads page](http://www.cloudera.com/downloads). 
+The driver is also available directly from [Simba technologies](http://www.simba.com/), who developed the driver.
+
+Virtual Schema Version| Hive Version | Driver Name | Driver Version 
+----------------------|--------------|-------------|-----------------
+ 4.0.3                | 2.3.2        | HiveJDBC    | 4.1
```
Comment (chiaradiamarcelo) [inline doc/user_guide/hive_user_guide.md:366]: Please also add here the information regarding the integration test and how to run/enable it

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,426 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>hive-virtual-schema</artifactId>
+    <version>1.0.0</version>
+    <name>Virtual Schema Hive</name>
```
Comment (chiaradiamarcelo) [inline pom.xml:6]:
```suggestion
    <name>Virtual Schema for Hive</name>
```

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,426 @@
...
+    </repositories>
+    <dependencies>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>virtual-schema-common-jdbc</artifactId>
+            <version>${vscjdbc.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>db-fundamentals-java</artifactId>
```
Comment (chiaradiamarcelo) [inline pom.xml:51]: this dependency is not necessary

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,426 @@
...
+            <version>0.1.1</version>
+        </dependency>
+        <!--Unit test dependencies-->
+        <dependency>
+            <groupId>nl.jqno.equalsverifier</groupId>
+            <artifactId>equalsverifier</artifactId>
```
Comment (chiaradiamarcelo) [inline pom.xml:57]: this dependency is not necessary

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,426 @@
...
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.5</version>
+                <executions>
+                    <execution>
+                        <id>prepare-agent</id>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+
```
Comment (chiaradiamarcelo) [inline pom.xml:253]:
```suggestion
```

File: pom.xml [hunk 8]
```diff
@@ -0,0 +1,426 @@
...
+                    </execution>
+                    
+                    <execution>
+                        <id>report</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+
```
Comment (chiaradiamarcelo) [inline pom.xml:261]:
```suggestion
```

File: pom.xml [hunk 9]
```diff
@@ -0,0 +1,426 @@
...
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>${surefire.and.failsafe.plugin.version}</version>
+                <configuration>
+                    <!-- Set the highest log level for coverage testing, so that we have a chance to reach branches
+                      <property>	                      in the logging lambdas too. -->
+                    <argLine>-Djava.util.logging.config.file=src/test/resources/logging.properties ${argLine}</argLine>
+                </configuration>
```
Comment (chiaradiamarcelo) [inline pom.xml:302]: shouldn't the integration test be disabled by default?

File: src/test/java/com/exasol/adapter/dialects/AbstractIntegrationTest.java [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+import java.sql.*;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.bucketfs.BucketAccessException;
+
+public abstract class AbstractIntegrationTest {
```
Comment (chiaradiamarcelo) [inline src/test/java/com/exasol/adapter/dialects/AbstractIntegrationTest.java:16]: this class makes no sense anymore. Please remove its content to src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectIT.java. Also change the `protected` modifiers by `private`

File: src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectTest.java [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.capabilities.Capabilities;
+import com.exasol.adapter.dialects.PropertyValidationException;
+import com.exasol.adapter.dialects.SqlDialect;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class HiveSqlDialectTest {
+    private SqlDialect dialect;
```
Comment (chiaradiamarcelo) [inline src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectTest.java:31]:
```suggestion
    private HiveSqlDialect dialect;
```

File: .travis.yml (hunk 1 above)
Comment (AnastasiiaSergienko) [reply .travis.yml:19]: This is not necessary I think, Sonar works fine without it, it a default value

File: pom.xml (hunk 9 above)
Comment (AnastasiiaSergienko) [reply pom.xml:302]: excluded

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**